circle.yml

```diff
@@ -1,16 +0,0 @@
-machine:
-  services:
-    - docker
-
-dependencies:
-  cache_directories:
-    - "~/docker"
-  override:
-    - if [[ -e ~/docker/image.tar ]]; then docker load -i ~/docker/image.tar; fi
-    - docker build -t moddep .
-    - mkdir -p ~/docker; docker save moddep > ~/docker/image.tar
```

Comment:
Do you think we should try to preserve the image caching?  I'm not sure how well it works in v2.0, without using the official docker layer caching (that costs extra money?)... and it seems that the build takes only a little over a minute in CircleCI right now, so I feel like it wouldn't save much time.  What do you think?

Reply:
I have no strong feelings here. I differ to your better judgement.